### PR TITLE
DOCSP-32898 Enable Natural Language Querying

### PR DESCRIPTION
**Pull request by ianf-mongodb**
## DESCRIPTION

Content for enabling the generative AI query feature for Compass.

## STAGING

https://docs-mongodbcom-staging.corp.mongodb.com/compass/docsworker-xlarge/DOCSP-32898/query-with-natural-language/enable-natural-language-querying/

https://docs-mongodbcom-staging.corp.mongodb.com/compass/docsworker-xlarge/DOCSP-32898/settings/settings-reference/#settings

## JIRA

https://jira.mongodb.org/browse/DOCSP-32898

## BUILD LOG

https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=650b11a3492df295cf23abbf

## Self-Review Checklist

- [ ] Is this free of any warnings or errors in the RST?
- [ ] Is this free of spelling errors?
- [ ] Is this free of grammatical errors?
- [ ] Is this free of staging / rendering issues?
- [ ] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: source/query-with-natural-language/enable-natural-language-querying.txt [hunk 1]
```diff
@@ -9,6 +9,86 @@ Enable Natural Language Querying
 .. contents:: On this page
    :local:
    :backlinks: none
-   :depth: 2
+   :depth: 1
+
+You can enable Compass natural language querying by toggling the 
+:guilabel:`AI Query Generator` setting. Natural language queries can
```
Comment (bsradcliffe) [inline source/query-with-natural-language/enable-natural-language-querying.txt:15]:
```suggestion
:guilabel:`Use Generative AI` setting. Natural language queries can 
```

File: source/query-with-natural-language/enable-natural-language-querying.txt [hunk 2]
```diff
@@ -9,6 +9,86 @@ Enable Natural Language Querying
...
+:guilabel:`Log in to Atlas to enable` button within the 
+:guilabel:`Use natural language to generate queries and pipelines` modal. 
+This modal displays after updating Compass to a version that supports 
+natural language querying and clicking the :guilabel:`Generate` 
+button on the query bar.
+
+Before you Begin
+----------------
+
+To enable Compass natural language querying you must connect to an
```
Comment (bsradcliffe) [inline source/query-with-natural-language/enable-natural-language-querying.txt:32]:
```suggestion
To enable the generation of queries and aggregations in Compass, you must connect to an 
```

File: source/query-with-natural-language/enable-natural-language-querying.txt [hunk 3]
```diff
@@ -9,6 +9,86 @@ Enable Natural Language Querying
...
+         `Atlas login page <https://account.mongodb.com/account/login>`_.
+
+      #. Complete the login procedure
+
+         Once completed a :guilabel:`Logged in with Atlas account`
+         message displays. 
+
+   .. step:: Enable the :guilabel:`AI Query Generator` feature
+
+      I. Click the toggle for :guilabel:`Use AI to generate queries with a
```
Comment (bsradcliffe) [inline source/query-with-natural-language/enable-natural-language-querying.txt:75]: At this point in time, a user is directed back from Atlas to Compass, where the "Data Processing Agreement" modal displays. When they accept it, the toggle is automatically activated, so there is no need for them to click the toggle.

File: source/query-with-natural-language/enable-natural-language-querying.txt [hunk 4]
```diff
@@ -9,6 +9,86 @@ Enable Natural Language Querying
...
+
+         A web browser displays the
+         `Atlas login page <https://account.mongodb.com/account/login>`_.
+
+      #. Complete the login procedure
+
+         Once completed a :guilabel:`Logged in with Atlas account`
+         message displays. 
+
+   .. step:: Enable the :guilabel:`AI Query Generator` feature
```
Comment (bsradcliffe) [inline source/query-with-natural-language/enable-natural-language-querying.txt:73]:
```suggestion
   .. step:: Enable the :guilabel:`Use Generative AI` feature
```

**Review by gribnoysup — APPROVED** [review]

File: source/query-with-natural-language/enable-natural-language-querying.txt (hunk 3 above)
Comment (ianf-mongodb) [reply source/query-with-natural-language/enable-natural-language-querying.txt]: Alright I have removed this from the steps.

**Pull request merged**